machines/openID4VP/openID4VPSelectors.ts

```diff
@@ -106,7 +106,10 @@ export function selectIsFaceVerifiedInVPSharing(state: State) {
 }
 
 export function selectVerifierNameInVPSharing(state: State) {
-  return state.context.authenticationResponse['client_metadata']?.name;
+  return (
+    state.context.authenticationResponse['client_metadata']?.['client_name'] ??
```

Comment:
client_name is this from spec?

Reply:
yes sreenadh
it is mentioned in the spec -> https://www.rfc-editor.org/rfc/rfc7591.txt 
please refer to page 9

Reply:
https://www.rfc-editor.org/rfc/rfc7591#section-2